scripts/v.import/v.import.py

```diff
@@ -93,8 +93,8 @@
 # %end
 # %flag
 # % key: o
-# % label: Override projection check (use current location's projection)
-# % description: Assume that the dataset has the same projection as the current location
+# % label: Override projection check (use current project's projection)
+# % description: Assume that the dataset has the same projection as the current project
```

Comment:
I know the changes are targeted to location-->project, but now project's projection sounds repetitive... can we change that to coordinate reference system or CRS?
```suggestion
# % label: Override projection check (use current project's CRS)
# % description: Assume that the dataset has the same coordinate reference system (CRS) as the current project
```